TypeSystem/Schema/golden/smoke_test_struct.cc

```diff
@@ -69,7 +69,7 @@ struct TemplatedInheriting_T9200000002835747520 : A {
 };
 struct Templated_T9209626390174323094 {
   int32_t foo;
-  TemplatedInheriting_T9200000002835747520 bar;
+  TemplatedInheriting_Z bar;
```

Comment:
Pure C++ schema exporter is unused now. It's missing another important piece: a base class with a virtual destructor for what were `CURRENT_STRUCT`-s.

My understanding was that we revisit this once we get into the high-performance `dlopen` territory.